padertorch/train/hooks.py

```diff
@@ -375,10 +377,18 @@ def pre_step(self, trainer: 'pt.Trainer'):
             self.finalize_summary(trainer)
             self.dump_summary(trainer)
 
+        # The check using the trigger doesn't work when the hook is loaded from
+        # a snapshot (i.e., set_last got invoked). Because of this, we have a
+        # flag that is set in reset_summary to determine when to compute
+        # snapshots
+        trainer.model.create_snapshot = self.create_snapshot
+        self.create_snapshot = False
```

Comment:
Can you change this to something like the following?
```python
if self.create_snapshot:
    trainer.model.create_snapshot = True
    self.create_snapshot = False
```
That would avoid a potential conflict between the hook.
e.g. Assign multiple summary hooks with different report intervals.

Reply:
Done